### PR TITLE
[release] 1.1.8: Remove deprecated-styles. Use react-leaflet context for markers.

### PR DESCRIPTION
**Pull request by YUzhva**
* Dependencies Update
* Demo-app: Update Panel props according to latest react-bootstrap specification.
* **Remove** requiring of **deprecated-styles** inside **react-leaflet-markercluster.js** plugin
* **Use react context store to access markers instead of cloning markers during their render.**

TODO: update documentation.

### CODE REVIEW

File: src/react-leaflet-markercluster.js [hunk 1]
```diff
@@ -60,7 +56,7 @@ export default class MarkerClusterGroup extends LayerGroup {
   componentWillReceiveProps(nextProps) {
     if (!isArraysEqual(this.props.markers, nextProps.markers)) {
       // Remove layer from map with previously rendered clustered markers
-      this.layerContainer.removeLayer(this.leafletElement);
+      // this.layerContainer.removeLayer(this.leafletElement);
```
Comment (rm-hull) [inline src/react-leaflet-markercluster.js]: Was this change intentional?

Comment (YUzhva) [reply src/react-leaflet-markercluster.js]: yep.

Because of removing `markers leaflet element` from `layerContainer` during component update `markers` was not added to the map, so you are adding/deleting markers from `Map` and instead of changes, all markers were removed.

Comment (rm-hull) [reply src/react-leaflet-markercluster.js]: :+1:

Maybe just get rid of the line altogether and the preceding comment then?
